projects/sandbox/datagen/datagen/scripts/timeslide_waveforms.py

```diff
@@ -47,7 +47,60 @@ def main(
     verbose: bool = False,
 ):
     """
-    Generates the waveforms for a single segment
+    Generates the waveforms for a single segment.
+
+    Args:
+        start:
+            GPS time of the beginning of the testing dataset
+        stop:
+            GPS time of the end of the testing dataset
+        shifts:
+            A list of shifts in seconds. Each value corresponds to the
+            the length of time by which an interferometer's timeseries is
+            moved during one shift. For example, if `ifos = ["H1", "L1"]`
+            and `shifts = [0, 1]`, then the Livingston timeseries will be
+            advanced by one second per shift, and Hanford won't be shifted
+        background:
+            Directory containing background data to use for PSD
+            calculation. Should have data for each interferometer in
+            the format generated by `background.py`
+        spacing:
+            The spacing between signals in seconds
+        buffer:
+            The buffer to apply to each side of the segment in seconds
+        prior:
+            A function that returns a Bilby PriorDict when called
+        minimum_frequency:
+            Minimum_frequency for generating waveform; not to
+            be confused with highpass filter frequency
+        reference_frequency:
+            Reference frequency for generating waveform
+        sample_rate:
+            Rate at which to sample time series
+        waveform_duration:
+            Duration of waveform in seconds
+        waveform_approximant:
+            Name of the waveform approximant to use.
+        highpass:
+            The frequency to use for a highpass filter
+        snr_threshold:
+            Minimum SNR of generated waveforms. Sampled parameters
+            that result in an SNR below this threshold will be rejected,
+            but saved for later use
+        ifos:
```

Comment:
Maybe make clear that this is expected to match the length of `shifts`. I would maybe have this higher up the arg list as well but maybe that's outside the scope of this PR